{{cookiecutter.project_slug}}/clients/vue3/src/App.vue

```diff
@@ -1,16 +1,36 @@
 <template>
-  <div id="nav">
-    <router-link to="/">Home</router-link>
-    <template v-if="isLoggedIn">
-      | <router-link to="/dashboard">Dashboard</router-link> |
-      <span id="logout" @click="logout()">Logout</span>
-    </template>
-    <template v-else>
-      | <router-link to="/login" data-cy="login">Login</router-link> |
-      <router-link to="/signup">Signup</router-link>
-    </template>
+  <div class="app font-avenir antialiased text-center">
+    <div id="nav" class="nav flex justify-center p-7">
```

Comment:
**Info:** Don't necessarily need the `app` or `nav` class names here, but added them in to help clarify the different functions of the different `<div>`'s

Reply:
Could: remove app and nav classes since they may be confusing to figure out mixed in with the other tailwind classes